components/hab/src/main.rs

```diff
@@ -1038,13 +1038,18 @@ fn sub_svc_load(m: &ArgMatches<'_>) -> Result<()> {
     Ok(())
 }
 
-fn sub_svc_unload(m: &ArgMatches<'_>) -> Result<()> {
+fn sub_svc_unload(m: &ArgMatches<'_>, feature_flags: FeatureFlag) -> Result<()> {
     let ident = PackageIdent::from_str(m.value_of("PKG_IDENT").unwrap())?;
     let cfg = config::load()?;
     let listen_ctl_addr = listen_ctl_addr_from_input(m)?;
     let secret_key = ctl_secret_key(&cfg)?;
-    let mut msg = protocol::ctl::SvcUnload::default();
-    msg.ident = Some(ident.into());
+
+    let timeout = maybe_get_shutdown_timeout(m, feature_flags)?.map(Into::into);
```

Comment:
🤔 That _is_ nicer.

Reply:
Well, hrmm... on second thought, there's a balancing act here. This is essentially adding type annotations, which we generally don't do unless we have to for the compiler's sake.

Do you have an idea on a general rule / guiding principle by which to determine whether we use `into` or `from`? Or why not something equivalent like 

```rust
let timeout: Option<u32> = maybe_get_shutdown_timeout(m, feature_flags)?.map(Into::into);
```

Reply:
My decision process is:
1. Avoid `Into::into` because the redundancy bothers me; any space that's being taken up in program text should convey meaningful info
1. If both `Foo:from` and `Bar::into` are possible, choose whichever provides more useful information in that particular context

I agree that these are like type annotations, and if I could eliminate them altogether, I'd probably prefer doing that since leaving information that isn't relevant out enhances simplicity, but at long as we have to say *something*, I prefer to say something meaningful.

Reply:
@baumanj That sounds good... I don't have a particularly strong opinion, but having a sensible rubric one way or the other is definitely better than nothing at all.

Now, if only we could automate this 😅

Reply:
@baumanj @christophermaier I went through the codebase and converted `Into::into` to `from`. Two inconveniences of the `from` version are:

1. If we do not use the name of the type we are converting into anywhere but the `from` statement we have to add a `use` statement we would not otherwise need.
1. If we change the name of the type we have to change the name in any `from` statements.

Not saying we should change our decision, but I think it is good to be aware of the tradeoffs.